src/c++/uds/userLinux/uds/logger.c

```diff
@@ -116,7 +116,7 @@ static void init_logger(void)
 	bool is_abs_path = false;
 	if (log_file != NULL) {
 		is_abs_path =
-			(make_abs_path(log_file, &log_file) == UDS_SUCCESS);
+			(make_abs_path(log_file, &log_file) == VDO_SUCCESS);
 		errno = 0;
```

Comment:
Don't change the error code here. The right check is probably "!= 0" but it's not related to this commit at any rate.